docs/source/API/core/policies/ThreadVectorMDRange.rst

```diff
@@ -1,52 +1,60 @@
-# `ThreadVectorMDRange`
+``ThreadVectorMDRange``
+=======================
 
-Header File: `Kokkos_Core.hpp`
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
```

Comment:
Please change `Usage` to `Description` and add a couple of explanatory sentences.